include/sys/dsl_scan.h

```diff
@@ -67,7 +68,9 @@ typedef struct dsl_scan_phys {
 	uint64_t scn_flags; /* dsl_scan_flags_t */
 } dsl_scan_phys_t;
 
+
 #define	SCAN_PHYS_NUMINTS (sizeof (dsl_scan_phys_t) / sizeof (uint64_t))
+#define	RESCRUB_BLOCK_IN_TXG 10000
```

Comment:
This shouldn't be a #define, this should be a tunable (see for example zfs_scan_vdev_limit).